scripts/testcafe_b.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+export CHROME_HEADLESS=true
+setup_service google-chrome-stable 83.0.4103.61-1
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+export TEST_SUITE_TYPE="junit"
+export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/junit"
+echo $TEST_SUITE_TYPE > $TEST_SUITE_TYPE_FILE
+echo $TEST_RESULT_FILE_DIR > $TEST_RESULT_FILE_DIR_FILE
+
+export SECOND_HALF=1   # only run second half of specs (in this suite)
```

Comment:
If this is the only difference, and the rest of the script is duplicated, do we run the risk of drifting if someone doesn't know to update both/all files? e.g., if the chrome version is different?